lib/object_store.rb

```diff
@@ -0,0 +1,39 @@
+module ObjectStore
+  class UnknownItemType < StandardError
+    def initialize(item_type)
+      super("Unknown item type: #{item_type}")
+    end
+  end
+
+  def self.configuration
+    @configuration ||= OpenStruct.new
+  end
+
+  def self.configure
+    yield(configuration)
```

Comment:
I wonder if we should validate the structure of is configuration so we can make certain assumptions about it, ensuring a certain structure and making sure that we don't accidentally break it.

Something like [Psych](https://docs.ruby-lang.org/en/master/Psych.html#module-Psych-label-Reading+from+a+file) to load the yml and check the structure? We should probably figure out a way to put that into the CI tests too.